c_src/device/bcm.c

```diff
@@ -227,8 +229,8 @@ int device_init(const device_opts_t* p_opts,
   uint32_t nvg_opts = 0;
   if (p_opts->antialias) nvg_opts |= NVG_ANTIALIAS;
   if (p_opts->debug_mode) nvg_opts |= NVG_DEBUG;
-  p_info->p_ctx = nvgCreateGLES2(nvg_opts);
-  if (p_info->p_ctx == NULL) {
+  p_info->v_ctx = nvgCreateGLES2(nvg_opts);
+  if (p_info->v_ctx == NULL) {
```

Comment:
```suggestion
  NVGcontext p_ctx = nvgCreateGLES2(nvg_opts);
  if (p_ctx == NULL) {
      log_error("RPI driver error: failed nvgCreateGLES2");
    return 0;
  }
  p_info->v_ctx = p_ctx;
```

Reply:
Makes sense (re https://github.com/ScenicFramework/scenic_driver_local/issues/48#issuecomment-1705606724). Updated in 37e1b7d982074af512ae06036103094a56157b6f